pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -39,21 +39,25 @@
 pd.options.mode.copy_on_write = True
 
 
-def read_population_data():
+def read_population_data(ref_year):
     """! Reads Population data from regionalstatistik.de
 
     A request is made to regionalstatistik.de and the StringIO is read in as a csv into the dataframe format.
+    @param ref_year [Default: 'newest'] or year (jjjj) convertible to str. Reference year.
     @return DataFrame
     """
-
-    download_url = 'https://www.regionalstatistik.de/genesis/online?operation=download&code=12411-02-03-4&option=csv'
+    if ref_year == 'newest':
+        download_url = 'https://www.regionalstatistik.de/genesis/online?operation=download&code=12411-02-03-4&option=csv'
+    else:
+        download_url = 'https://www.regionalstatistik.de/genesis/online?operation=download&code=12411-02-03-4&option=csv&zeiten=' + \
+            str(ref_year)
```

Comment:
For years with non available data pandas throws a parserError. I changed it, so that if this parserError is catched, the newest data will be downloaded (with a Warning).